src/test/MintRewardNotifierTestBase.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// slither-disable-start reentrancy-benign
+
+pragma solidity ^0.8.23;
+
+import {MintRewardNotifier} from "../notifiers/MintRewardNotifier.sol";
+import {StakerTestBase} from "./StakerTestBase.sol";
+
+/// @notice Base contract for testing MintRewardNotifier functionality.
+/// @dev Extends StakerTestBase and provides reward notification testing utilities.
+/// Simulates calling notify function on MintRewardNotifier.
```

Comment:
Better natspec, simulate is probably not the right term. We probably should explain where and when this should be used.